tests/DishwasherMaster.tex

```diff
@@ -0,0 +1,85 @@
+\section{Sp{\"u}lmaschinemeister [DSPL \& OPL]}
+The robot helps by placing all dirty dishes inside the dishwasher.
+
+\subsection{Goal}
+The robot has to correctly identify and manipulate cutlery, kitchenware, and tableware, taking them from the sink and safely placing them in the dishwasher rack, as well as leave the dishwasher ready to be set in motion.
+
+\subsection{Focus}
+This test focuses on object recognition and manipulation, paying particular attention in orientating objects for an adequate placement in narrow spaces.
+
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Dishwasher:} The dishwasher has its door closed and some utensils already inside.
+	\item \textbf{Location:} This test takes place in the kitchen of the arena.
+	\item \textbf{Sink:} The sink has 12 utensils (See \ref{rule:tableware_kitchenware}) in it. There are at least three dishes (or bowls) three mugs (or glasses, or cups), two cooking pan (or pot), and two spoons (or any other cutlery).
+\end{enumerate}
+
+
+\subsection{Task}
+The robot must execute the following tasks in any order as long as there are utensils in the sink.
+\begin{itemize}
+	\item \textbf{Place cutlery:} The robot takes any cutlery from the sink and safely places it in the cutlery basket of the dishwasher.
+
+	\item \textbf{Place cooking pans:} The robot takes any cooking pan (or pot, or utensil) from the sink and safely places it in the the dishwasher.
+
+	\item \textbf{Place dishes and bowls:} The robot takes any dish (or bowl) from the sink and safely places it in the corresponding rack in the dishwasher.
+
+	\item \textbf{Place glasses, mugs, and cups:} The robot takes any glass (or cup, or mug) from the sink and safely places it in the corresponding rack in the dishwasher.
+
+	\item \textbf{Place soap tab:} The robot takes a dishwasher soap tab from the bag in a nearby counter and places it inside the corresponding compartment in the dishwasher. The robot may ask for help to open and close the tab compartment.
+\end{itemize}
+
+Additionally, the robot may execute the following tasks to score extra points.
+\begin{itemize}
+	\item \textbf{Opening the dishwasher:} When requested by the team, the robot can open the dishwasher door before start to placing objects inside.
+
+	\item \textbf{Opening the dishwasher:} The robot must open the tab compartment door before placing the dishwasher soap tab.
+
+	\item \textbf{Closing the dishwasher:} The robot must close the tab compartment door immediately after placing the dishwasher soap tab.
```

Comment:
This is very very difficult. Bonus points for that

Reply:
50 points each according with the scoresheet, so I'm not fixing it. In fact, dealing with the tab grants 160 points, which is enough IMO.

If TC reaches consensus on higher scoring, it can be easily updated.